### PR TITLE
Add selenium tests for alert pages #34

### LINKED ISSUE #34: Add selenium tests for alert pages
Similar to how the tests in `selenium-tests/tests` have tests for various pages of our site, we should add tests for the alert pages:
- all-alerts.html
- alert.html
- contacts.html

See the selenium tests readme (which should be added by https://github.com/siglens/siglens/issues/33) for information on getting started with writing selenium tests.

### PR DESCRIPTION
**Pull request by shyam2520**
# Description
Wrote test cases for alerts.html page to check if the elements in the page exist and if so are they editable and clickable. 

Fixes #34(https://github.com/siglens/siglens/issues/34)

# Testing
Induced bugs in alerts.html to check if the selenium script runs catches those, reviewers can comment on any element in the page to check if the script catches or not. 

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

**Comment by nkunal** [comment]
@sonamgupta21 please review

File: selenium-tests/tests/alerts-test.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+//   }
+// })();
+
+const { By, Key, Builder, WebElement } = require("selenium-webdriver");
+const assert = require("assert");
+const chrome = require('selenium-webdriver/chrome');
+let driver;
+const chromeOptions = new chrome.Options()
+
+// chromeOptions.addArguments('--headless');
```
Comment (sonamgupta21) [inline selenium-tests/tests/alerts-test.js:21]: Please uncomment this line.

File: selenium-tests/tests/alerts-test.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
+// const {Builder, By, Key, until} = require('selenium-webdriver');
+
+// (async function example() {
```
Comment (sonamgupta21) [inline selenium-tests/tests/alerts-test.js:3]: Please remove the commented code.

Comment (shyam2520) [reply selenium-tests/tests/alerts-test.js]: I have made the changes for the same

File: selenium-tests/tests/alerts-test.js (hunk 1 above)
Comment (shyam2520) [reply selenium-tests/tests/alerts-test.js]: I have made the changes for the same

**Comment by sonamgupta21** [comment]
Looks good!
@shyam2520 Please update your branch.

**Comment by shyam2520** [comment]
I have updated the branch

**Review by sonamgupta21 — APPROVED** [review]

**Pull request merged**